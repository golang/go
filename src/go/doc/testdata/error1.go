// Copyright 2012 The Go Authors. All rights reserved.
// Use of this source code is governed by a BSD-style
// license that can be found in the LICENSE file.

package error1

type I0 interface {
	// When embedded, the predeclared error interface
	// must remain visible in interface types.
	error
}

type T0 struct {
	ExportedField interface {
		// error should be visible
		error
	}
}

type S0 struct {
	// In struct types, an embedded error must only be visible
	// if AllDecls is set.
	error
}
