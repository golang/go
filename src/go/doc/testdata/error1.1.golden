// 
PACKAGE error1

IMPORTPATH
	testdata/error1

FILENAMES
	testdata/error1.go

TYPES
	// 
	type I0 interface {
		// When embedded, the predeclared error interface
		// must remain visible in interface types.
		error
	}

	// 
	type S0 struct {
		// In struct types, an embedded error must only be visible
		// if AllDecls is set.
		error
	}

	// 
	type T0 struct {
		ExportedField interface {
			// error should be visible
			error
		}
	}

